### PR TITLE
Add opengraph/dom-tests

### PR DESCRIPTION
**Pull request by b4handjr**
r? @silne30 @rpappalax 

I know there is still a manifest file that needs to be made but this is a start.

### CODE REVIEW

File: opengraph/dom-test/test_opengraph_standards.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+            for item in meta:
+                if item.get('name') == req_tags or item.get('property') == req_tags:
+                    found_tags.append(req_tags)
+                else:
+                    continue
+            # Test found tags against required
+            if req_tags in found_tags:
+                continue
+            else:
+                print "Item {} not found".format(req_tags)
```
Comment (rpappalax) [inline opengraph/dom-test/test_opengraph_standards.py]: this should be python3 compliant.  change to:
 print("Item {0} not found".format(req_tags))

File: opengraph/dom-test/conftest.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# Load url and parse it
+@pytest.fixture
+def meta(response_obj):
+    soup = BeautifulSoup(response_obj.content, "html.parser")
+    return soup.find_all("meta")
+
+
+@pytest.fixture
+def response_obj():
+    return requests.get('http://nightly.mozilla.org')
```
Comment (rpappalax) [inline opengraph/dom-test/conftest.py]: will this always be tested against this URL or will it change?
if so, consider defining a constant for it.

File: opengraph/dom-test/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# OpenGraph Standards test
```
Comment (rpappalax) [inline opengraph/dom-test/README.md:1]: I'd add a few more sections to the README 

## Requirements

all requirements will be installed by the run file (see Test Execution section below) and assume a Linux test environment.
This test runs against remote URLs and doesn't require a browser.

## Test Execution

```
$ cd services-test/opengraph/dom-test
$ run
```

File: opengraph/dom-test/conftest.py (hunk 2 above)
Comment (rpappalax) [inline opengraph/dom-test/conftest.py]: Per our conversation, you'll need to pull in and loop through a list of all the sites you want to test (not just this one URL).  That list should ideally reside in a config file instead.

Comment (rpappalax) [inline opengraph/dom-test/conftest.py]: something like:

```
for url in config_urls:
    url = 'https://{0}'.format(url)
```

**Comment by chartjes** [comment]
Thanks @jrbenny35

**Pull request merged**